### PR TITLE
fix typecheck error in main

### PR DESCRIPTION
**Pull request by wimax-grapl**
```
src/python/graplctl/graplctl/cli.py:27: error: Argument 1 to "Choice" has incompatible type "KeysView[str]"; expected "Sequence[str]"  [arg-type]
```

Sequence implies you can `seq[idx]` on it

### CODE REVIEW

**Review by jgrillo-grapl — APPROVED** [review]
Could we pin our click dep as well?

**Review by jgrillo-grapl — APPROVED** [review]
yolo

File: 3rdparty/python/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# requirements.txt
+Declare all your dependencies.
+Ideally, we'd pin everything in here, but it's not the end of the universe if a few are unpinned.
```
Comment (christophermaier) [inline 3rdparty/python/README.md:3]: Things should probably only be pinned in here if they need to be for some reason (and that reason should be documented in a comment).

Comment (wimax-grapl) [reply 3rdparty/python/README.md]: updating readme

**Pull request merged**